test/cluster-tests/dcos/marathon-slave-public.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
is this the same as the one that already exists? just curious from a consistency standpoint.